.travis.yml

```diff
@@ -39,7 +39,7 @@ script:
   - ./run_tests.py --outofsource --fast hybrid
   # install cvxopt.glpk
   - export CVXOPT_BUILD_GLPK=1
-  - pip install cvxopt==1.1.8
+  - pip install cvxopt==1.2.0
```

Comment:
[On Travis CI `polytope` installs the latest `cvxopt` version](https://github.com/tulip-control/polytope/blob/ccb080dbae7ffbb46e2ebe016665d9d0895fc1cd/.travis.yml#L43) and [in `requirements/extras.txt` `polytope` specifies `cvxopt==1.2.4`](https://github.com/tulip-control/polytope/blob/ccb080dbae7ffbb46e2ebe016665d9d0895fc1cd/requirements/extras.txt#L1). The latest `cvxopt` version is [1.2.5](https://pypi.org/project/cvxopt/#history). A number of possibilities would be to unpin `cvxopt` here, or pin `cvxopt` for `polytope` on Travis CI.

Reply:
It seems most reasonable to let `polytope` decide what version of `cvxopt` that should be installed. After all, overriding `polytope`'s requirements led to this problem in the first place.

So, removing lines 41 and 42 entirely would make pip look at the requirements of `polytope`? Do we risk not getting GLPK in that case?

Reply:
`cvxopt` is not listed in the requirements of [`polytope`'s `setup.py`](https://github.com/tulip-control/polytope/blob/ccb080dbae7ffbb46e2ebe016665d9d0895fc1cd/setup.py) because it is not a default requirement. So when `pip` installs `polytope` it will not automatically install `cvxopt`, and lines 41 and 42 would be needed. If the latest version of `cvxopt` is installed here, then line 42 could become `pip install cvxopt`.

Reply:
Right, I am learning of how `pip`, Travis and `polytope` are set up as we go. Using latest version or hard-coding it to the same version as `polytope` has right now would solve the problem, but it will open up for the same problem again in the future. We could pin the versions of both `polytope` and `cvxopt`, but then we put unnecessary restrictions on users of TuLiP who do not use `cvxopt`. The ideal solution would be to fetch the correct version of `extras.txt` and feed it to `pip`. In that way `polytope` defines which version of `cvxopt` to use, and we do not need to keep track of the correct version in TuLiP too.